packages/ui/src/helpers/authenticator/__tests__/defaultAuthHubHandler.test.ts

```diff
@@ -0,0 +1,108 @@
+import { HubCapsule } from '@aws-amplify/core';
```

Comment:
As mentioned in PR description, moved a lot of `listenToAuthHub` tests to this new file to test `defaultAuthHubHandler` directly instead of indirectly through `listenToAuthHub`.